app/validation/validators.py

```diff
@@ -75,12 +75,12 @@ def __call__(self, form, field):
         error_message = None
         if data is not None:
             if self.minimum is not None and data < self.minimum:
-                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_SMALL'])
+                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_SMALL']) % dict(min=self.minimum, max=self.maximum)
```

Comment:
Do we really need to cover both format methods? We could remove the % dict from this line and line 80 and put it back in on line 83 where the error is raised.